public/js/search.js

```diff
@@ -15,6 +15,8 @@
       this.element.on('click', '#news-tab a.page-next', $.proxy(this._onNewsNextPageClick, this));
       this.element.on('click', '#files-tab a.page-prev', $.proxy(this._onFilesPrevPageClick, this));
       this.element.on('click', '#files-tab a.page-next', $.proxy(this._onFilesNextPageClick, this));
+      this.element.on("click", "#wordpress-pages-tab a.page-prev", this.$.proxy(this._onWordpressPagesPrevPageClick, this));
```

Comment:
Maybe this should just be called "oppiminen" instead of Wordpress? (applies to all following changes)

Reply:
"wordpress" changed to "oppiminen" everywhere in the code